.github/workflows/publish.yml

```diff
@@ -40,7 +28,7 @@ jobs:
           access: public
       - name: Deploy
         # merge only if version is greater than the one in npm and only after merge
-        if: steps.package_version.outputs.npm_is_greater == 'false'
+        if: steps.publish.outputs.type == 'none'
```

Comment:
```suggestion
        if: steps.publish.outputs.type != 'none'
```

?

Reply:
Thank you. Do to the wrong placed id this was working erroneously. Corrected.